Formula/ldid.rb

```diff
@@ -17,7 +17,7 @@ class Ldid < Formula
   depends_on "openssl"
 
   def install
-    inreplace "./make.sh", /^.*\/Applications\/Xcode-5.1.1.app.*/, ""
+    inreplace "./make.sh", %r{/^.*\/Applications\/Xcode-5.1.1.app.*/}, ""
```

Comment:
Ah, I see the misunderstanding here. Regexes are wrapped by either `%r{}` or `//`, not both. Where there's a nudge to use `%r{}` you can drop the `//`. You can also drop any escaped forward slashes, which aren't necessary inside `%r{}`-style regexes. In this case, the change would reflect:
```ruby
%r{^.*/Applications/Xcode-5.1.1.app.*}
```